src/server/ocpp/services/OCPPService.ts

```diff
@@ -824,6 +837,8 @@ export default class OCPPService {
             }
             // Flag
             lastTransaction.stop.extraInactivityComputed = true;
+            // Billing - We now know the extra inactivity - the invoice can be generated
+            await OCPPUtils.processTransactionBilling(tenant, lastTransaction, TransactionAction.END);
```

Comment:
Possible to put this call just before the first CDR call in line 845?

Reply:
Be sure that you handle this use case in the billing ;-)

Reply:
If it throws an exception during a Status Notification it has no effect on the OCPP return call (always return an empty response).
In the end you will have only a log in error.
Whether add a try...catch and log a warning in the caller or remove the exception throw in the billing and log a warning.
As you want.

Reply:
We can discuss this tomorrow onsite ;-)